Bug Report: Support Bundle can not be created with MongoDB v8.2

<!--- Provide a general summary of the issue in the Title above -->

## Expected Behavior
Using MongoDB v7 Support Bundle can be created through the WebUI without any issues or error messages.

## Current Behavior
Using MongoDB v8 no Support Bundle can be created through the WebUI. 

These are the Error Messages: 

- in the UI
<img width="1368" height="833" alt="Image" src="https://github.com/user-attachments/assets/51ea4103-23b0-4908-818a-a10abd4be1f2" />

- in the logs
[graylog.log](https://github.com/user-attachments/files/24237679/graylog.log)

`graylog1-1  | 2025-12-18 14:17:19,687 WARN : org.glassfish.jersey.internal.Errors - The following warnings have been detected: WARNING: Unknown HK2 failure detected:
graylog1-1  | MultiException stack 1 of 1
graylog1-1  | com.google.inject.ProvisionException: Unable to provision, see the following errors:
graylog1-1  | 
graylog1-1  | 1) [Guice/ErrorInjectingConstructor]: NullPointerException: Null sysInfo
graylog1-1  |   at MongoProbe.<init>(MongoProbe.java:53)
graylog1-1  |   at MongoProbe.class(MongoProbe.java:43)
graylog1-1  |   at ClusterStatsService.<init>(ClusterStatsService.java:51)
graylog1-1  |       \_ for 1st parameter
graylog1-1  |   at ClusterStatsService.class(ClusterStatsService.java:51)
graylog1-1  |   at SupportBundleService.<init>(SupportBundleService.java:144)
graylog1-1  |       \_ for 7th parameter
graylog1-1  |   while locating SupportBundleService
graylog1-1  | 
graylog1-1  | Learn more:
graylog1-1  |   https://github.com/google/guice/wiki/ERROR_INJECTING_CONSTRUCTOR
graylog1-1  | 
graylog1-1  | 1 error
graylog1-1  | 
graylog1-1  | ======================
graylog1-1  | Full classname legend:
graylog1-1  | ======================
graylog1-1  | ClusterStatsService:  "org.graylog2.system.stats.ClusterStatsService"
graylog1-1  | MongoProbe:           "org.graylog2.system.stats.mongo.MongoProbe"
graylog1-1  | SupportBundleService: "org.graylog2.rest.resources.system.debug.bundle.SupportBundleService"
graylog1-1  | ========================
graylog1-1  | End of classname legend:
graylog1-1  | ========================
graylog1-1  | 
graylog1-1  | 	at com.google.inject.internal.InternalProvisionException.toProvisionException(InternalProvisionException.java:251)
graylog1-1  | 	at com.google.inject.internal.InjectorImpl$1.get(InjectorImpl.java:1151)
graylog1-1  | 	at org.jvnet.hk2.guice.bridge.internal.GuiceServiceHk2Bean.create(GuiceServiceHk2Bean.java:90)
graylog1-1  | 	at org.jvnet.hk2.internal.SystemDescriptor.create(SystemDescriptor.java:471)
graylog1-1  | 	at org.jvnet.hk2.guice.bridge.internal.GuiceScopeContext.findOrCreate(GuiceScopeContext.java:51)
graylog1-1  | 	at org.jvnet.hk2.internal.Utilities.createService(Utilities.java:2111)
graylog1-1  | 	at org.jvnet.hk2.internal.ServiceLocatorImpl.getService(ServiceLocatorImpl.java:683)
graylog1-1  | 	at org.jvnet.hk2.internal.ThreeThirtyResolver.resolve(ThreeThirtyResolver.java:55)
graylog1-1  | 	at org.jvnet.hk2.internal.ClazzCreator.resolve(ClazzCreator.java:197)
graylog1-1  | 	at org.jvnet.hk2.internal.ClazzCreator.resolveAllDependencies(ClazzCreator.java:214)
graylog1-1  | 	at org.jvnet.hk2.internal.ClazzCreator.create(ClazzCreator.java:387)
graylog1-1  | 	at org.jvnet.hk2.internal.SystemDescriptor.create(SystemDescriptor.java:479)
graylog1-1  | 	at org.glassfish.jersey.inject.hk2.RequestContext.findOrCreate(RequestContext.java:59)
graylog1-1  | 	at org.jvnet.hk2.internal.Utilities.createService(Utilities.java:2111)
graylog1-1  | 	at org.jvnet.hk2.internal.ServiceLocatorImpl.internalGetService(ServiceLocatorImpl.java:775)
graylog1-1  | 	at org.jvnet.hk2.internal.ServiceLocatorImpl.internalGetService(ServiceLocatorImpl.java:738)
graylog1-1  | 	at org.jvnet.hk2.internal.ServiceLocatorImpl.getService(ServiceLocatorImpl.java:708)
graylog1-1  | 	at org.glassfish.jersey.inject.hk2.AbstractHk2InjectionManager.getInstance(AbstractHk2InjectionManager.java:160)
graylog1-1  | 	at org.glassfish.jersey.inject.hk2.ImmediateHk2InjectionManager.getInstance(ImmediateHk2InjectionManager.java:30)
graylog1-1  | 	at org.glassfish.jersey.internal.inject.Injections.getOrCreate(Injections.java:122)
graylog1-1  | 	at org.glassfish.jersey.server.model.MethodHandler$ClassBasedMethodHandler.getInstance(MethodHandler.java:260)
graylog1-1  | 	at org.glassfish.jersey.server.internal.routing.PushMethodHandlerRouter.apply(PushMethodHandlerRouter.java:51)
graylog1-1  | 	at org.glassfish.jersey.server.internal.routing.RoutingStage._apply(RoutingStage.java:86)
graylog1-1  | 	at org.glassfish.jersey.server.internal.routing.RoutingStage._apply(RoutingStage.java:89)
graylog1-1  | 	at org.glassfish.jersey.server.internal.routing.RoutingStage._apply(RoutingStage.java:89)
graylog1-1  | 	at org.glassfish.jersey.server.internal.routing.RoutingStage._apply(RoutingStage.java:89)
graylog1-1  | 	at org.glassfish.jersey.server.internal.routing.RoutingStage._apply(RoutingStage.java:89)
graylog1-1  | 	at org.glassfish.jersey.server.internal.routing.RoutingStage.apply(RoutingStage.java:69)
graylog1-1  | 	at org.glassfish.jersey.server.internal.routing.RoutingStage.apply(RoutingStage.java:38)
graylog1-1  | 	at org.glassfish.jersey.process.internal.Stages.process(Stages.java:173)
graylog1-1  | 	at org.glassfish.jersey.server.ServerRuntime$1.run(ServerRuntime.java:266)
graylog1-1  | 	at org.glassfish.jersey.internal.Errors$1.call(Errors.java:248)
graylog1-1  | 	at org.glassfish.jersey.internal.Errors$1.call(Errors.java:244)
graylog1-1  | 	at org.glassfish.jersey.internal.Errors.process(Errors.java:292)
graylog1-1  | 	at org.glassfish.jersey.internal.Errors.process(Errors.java:274)
graylog1-1  | 	at org.glassfish.jersey.internal.Errors.process(Errors.java:244)
graylog1-1  | 	at org.glassfish.jersey.process.internal.RequestScope.runInScope(RequestScope.java:266)
graylog1-1  | 	at org.glassfish.jersey.server.ServerRuntime.process(ServerRuntime.java:253)
graylog1-1  | 	at org.glassfish.jersey.server.ApplicationHandler.handle(ApplicationHandler.java:696)
graylog1-1  | 	at org.glassfish.jersey.grizzly2.httpserver.GrizzlyHttpContainer.service(GrizzlyHttpContainer.java:367)
graylog1-1  | 	at org.glassfish.grizzly.http.server.HttpHandler$1.run(HttpHandler.java:190)
graylog1-1  | 	at com.codahale.metrics.InstrumentedExecutorService$InstrumentedRunnable.run(InstrumentedExecutorService.java:259)
graylog1-1  | 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
graylog1-1  | 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
graylog1-1  | 	at java.base/java.lang.Thread.run(Unknown Source)
graylog1-1  | Caused by: java.lang.NullPointerException: Null sysInfo
graylog1-1  | 	at org.graylog2.system.stats.mongo.AutoValue_BuildInfo.<init>(AutoValue_BuildInfo.java:58)
graylog1-1  | 	at org.graylog2.system.stats.mongo.BuildInfo.create(BuildInfo.java:80)
graylog1-1  | 	at org.graylog2.system.stats.mongo.MongoProbe.createBuildInfo(MongoProbe.java:118)
graylog1-1  | 	at org.graylog2.system.stats.mongo.MongoProbe.<init>(MongoProbe.java:59)
graylog1-1  | 	at org.graylog2.system.stats.mongo.MongoProbe.<init>(MongoProbe.java:53)
graylog1-1  | 	at org.graylog2.system.stats.mongo.MongoProbe$$FastClassByGuice$$ffffffff8c038ed6.GUICE$TRAMPOLINE(<generated>)
graylog1-1  | 	at org.graylog2.system.stats.mongo.MongoProbe$$FastClassByGuice$$ffffffff8c038ed6.apply(<generated>)
graylog1-1  | 	at com.google.inject.internal.DefaultConstructionProxyFactory$FastClassProxy.newInstance(DefaultConstructionProxyFactory.java:82)
graylog1-1  | 	at com.google.inject.internal.ConstructorInjector.provision(ConstructorInjector.java:114)
graylog1-1  | 	at com.google.inject.internal.ConstructorInjector.construct(ConstructorInjector.java:91)
graylog1-1  | 	at com.google.inject.internal.ConstructorBindingImpl$Factory.get(ConstructorBindingImpl.java:300)
graylog1-1  | 	at com.google.inject.internal.ProviderToInternalFactoryAdapter.get(ProviderToInternalFactoryAdapter.java:40)
graylog1-1  | 	at com.google.inject.internal.SingletonScope$1.get(SingletonScope.java:169)
graylog1-1  | 	at com.google.inject.internal.InternalFactoryToProviderAdapter.get(InternalFactoryToProviderAdapter.java:45)
graylog1-1  | 	at com.google.inject.internal.SingleParameterInjector.inject(SingleParameterInjector.java:40)
graylog1-1  | 	at com.google.inject.internal.SingleParameterInjector.getAll(SingleParameterInjector.java:60)
graylog1-1  | 	at com.google.inject.internal.ConstructorInjector.provision(ConstructorInjector.java:113)
graylog1-1  | 	at com.google.inject.internal.ConstructorInjector.construct(ConstructorInjector.java:91)
graylog1-1  | 	at com.google.inject.internal.ConstructorBindingImpl$Factory.get(ConstructorBindingImpl.java:300)
graylog1-1  | 	at com.google.inject.internal.ProviderToInternalFactoryAdapter.get(ProviderToInternalFactoryAdapter.java:40)
graylog1-1  | 	at com.google.inject.internal.SingletonScope$1.get(SingletonScope.java:169)
graylog1-1  | 	at com.google.inject.internal.InternalFactoryToProviderAdapter.get(InternalFactoryToProviderAdapter.java:45)
graylog1-1  | 	at com.google.inject.internal.SingleParameterInjector.inject(SingleParameterInjector.java:40)
graylog1-1  | 	at com.google.inject.internal.SingleParameterInjector.getAll(SingleParameterInjector.java:60)
graylog1-1  | 	at com.google.inject.internal.ConstructorInjector.provision(ConstructorInjector.java:113)
graylog1-1  | 	at com.google.inject.internal.ConstructorInjector.construct(ConstructorInjector.java:91)
graylog1-1  | 	at com.google.inject.internal.ConstructorBindingImpl$Factory.get(ConstructorBindingImpl.java:300)
graylog1-1  | 	at com.google.inject.internal.InjectorImpl$1.get(InjectorImpl.java:1148)
graylog1-1  | 	... 43 more
graylog1-1  | 
graylog1-1  | WARNING: Unknown HK2 failure detected:
graylog1-1  | MultiException stack 1 of 3
graylog1-1  | com.google.inject.ProvisionException: Unable to provision, see the following errors:
graylog1-1  | 
graylog1-1  | 1) [Guice/ErrorInjectingConstructor]: NullPointerException: Null sysInfo
graylog1-1  |   at MongoProbe.<init>(MongoProbe.java:53)
graylog1-1  |   at MongoProbe.class(MongoProbe.java:43)
graylog1-1  |   at ClusterStatsService.<init>(ClusterStatsService.java:51)
graylog1-1  |       \_ for 1st parameter
graylog1-1  |   at ClusterStatsService.class(ClusterStatsService.java:51)
graylog1-1  |   at SupportBundleService.<init>(SupportBundleService.java:144)
graylog1-1  |       \_ for 7th parameter
graylog1-1  |   while locating SupportBundleService
graylog1-1  | 
graylog1-1  | Learn more:
graylog1-1  |   https://github.com/google/guice/wiki/ERROR_INJECTING_CONSTRUCTOR
graylog1-1  | 
graylog1-1  | 1 error
graylog1-1  | 
graylog1-1  | ======================
graylog1-1  | Full classname legend:
graylog1-1  | ======================
graylog1-1  | ClusterStatsService:  "org.graylog2.system.stats.ClusterStatsService"
graylog1-1  | MongoProbe:           "org.graylog2.system.stats.mongo.MongoProbe"
graylog1-1  | SupportBundleService: "org.graylog2.rest.resources.system.debug.bundle.SupportBundleService"
graylog1-1  | ========================
graylog1-1  | End of classname legend:
graylog1-1  | ========================
graylog1-1  | 
graylog1-1  | 	at com.google.inject.internal.InternalProvisionException.toProvisionException(InternalProvisionException.java:251)
graylog1-1  | 	at com.google.inject.internal.InjectorImpl$1.get(InjectorImpl.java:1151)
graylog1-1  | 	at org.jvnet.hk2.guice.bridge.internal.GuiceServiceHk2Bean.create(GuiceServiceHk2Bean.java:90)
graylog1-1  | 	at org.jvnet.hk2.internal.SystemDescriptor.create(SystemDescriptor.java:471)
graylog1-1  | 	at org.jvnet.hk2.guice.bridge.internal.GuiceScopeContext.findOrCreate(GuiceScopeContext.java:51)
graylog1-1  | 	at org.jvnet.hk2.internal.Utilities.createService(Utilities.java:2111)
graylog1-1  | 	at org.jvnet.hk2.internal.ServiceLocatorImpl.getService(ServiceLocatorImpl.java:683)
graylog1-1  | 	at org.jvnet.hk2.internal.ThreeThirtyResolver.resolve(ThreeThirtyResolver.java:55)
graylog1-1  | 	at org.jvnet.hk2.internal.ClazzCreator.resolve(ClazzCreator.java:197)
graylog1-1  | 	at org.jvnet.hk2.internal.ClazzCreator.resolveAllDependencies(ClazzCreator.java:214)
graylog1-1  | 	at org.jvnet.hk2.internal.ClazzCreator.create(ClazzCreator.java:387)
graylog1-1  | 	at org.jvnet.hk2.internal.SystemDescriptor.create(SystemDescriptor.java:479)
graylog1-1  | 	at org.glassfish.jersey.inject.hk2.RequestContext.findOrCreate(RequestContext.java:59)
graylog1-1  | 	at org.jvnet.hk2.internal.Utilities.createService(Utilities.java:2111)
graylog1-1  | 	at org.jvnet.hk2.internal.ServiceLocatorImpl.internalGetService(ServiceLocatorImpl.java:775)
graylog1-1  | 	at org.jvnet.hk2.internal.ServiceLocatorImpl.internalGetService(ServiceLocatorImpl.java:738)
graylog1-1  | 	at org.jvnet.hk2.internal.ServiceLocatorImpl.getService(ServiceLocatorImpl.java:708)
graylog1-1  | 	at org.glassfish.jersey.inject.hk2.AbstractHk2InjectionManager.getInstance(AbstractHk2InjectionManager.java:160)
graylog1-1  | 	at org.glassfish.jersey.inject.hk2.ImmediateHk2InjectionManager.getInstance(ImmediateHk2InjectionManager.java:30)
graylog1-1  | 	at org.glassfish.jersey.internal.inject.Injections.getOrCreate(Injections.java:122)
graylog1-1  | 	at org.glassfish.jersey.server.model.MethodHandler$ClassBasedMethodHandler.getInstance(MethodHandler.java:260)
graylog1-1  | 	at org.glassfish.jersey.server.internal.routing.PushMethodHandlerRouter.apply(PushMethodHandlerRouter.java:51)
graylog1-1  | 	at org.glassfish.jersey.server.internal.routing.RoutingStage._apply(RoutingStage.java:86)
graylog1-1  | 	at org.glassfish.jersey.server.internal.routing.RoutingStage._apply(RoutingStage.java:89)
graylog1-1  | 	at org.glassfish.jersey.server.internal.routing.RoutingStage._apply(RoutingStage.java:89)
graylog1-1  | 	at org.glassfish.jersey.server.internal.routing.RoutingStage._apply(RoutingStage.java:89)
graylog1-1  | 	at org.glassfish.jersey.server.internal.routing.RoutingStage._apply(RoutingStage.java:89)
graylog1-1  | 	at org.glassfish.jersey.server.internal.routing.RoutingStage.apply(RoutingStage.java:69)
graylog1-1  | 	at org.glassfish.jersey.server.internal.routing.RoutingStage.apply(RoutingStage.java:38)
graylog1-1  | 	at org.glassfish.jersey.process.internal.Stages.process(Stages.java:173)
graylog1-1  | 	at org.glassfish.jersey.server.ServerRuntime$1.run(ServerRuntime.java:266)
graylog1-1  | 	at org.glassfish.jersey.internal.Errors$1.call(Errors.java:248)
graylog1-1  | 	at org.glassfish.jersey.internal.Errors$1.call(Errors.java:244)
graylog1-1  | 	at org.glassfish.jersey.internal.Errors.process(Errors.java:292)
graylog1-1  | 	at org.glassfish.jersey.internal.Errors.process(Errors.java:274)
graylog1-1  | 	at org.glassfish.jersey.internal.Errors.process(Errors.java:244)
graylog1-1  | 	at org.glassfish.jersey.process.internal.RequestScope.runInScope(RequestScope.java:266)
graylog1-1  | 	at org.glassfish.jersey.server.ServerRuntime.process(ServerRuntime.java:253)
graylog1-1  | 	at org.glassfish.jersey.server.ApplicationHandler.handle(ApplicationHandler.java:696)
graylog1-1  | 	at org.glassfish.jersey.grizzly2.httpserver.GrizzlyHttpContainer.service(GrizzlyHttpContainer.java:367)
graylog1-1  | 	at org.glassfish.grizzly.http.server.HttpHandler$1.run(HttpHandler.java:190)
graylog1-1  | 	at com.codahale.metrics.InstrumentedExecutorService$InstrumentedRunnable.run(InstrumentedExecutorService.java:259)
graylog1-1  | 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
graylog1-1  | 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
graylog1-1  | 	at java.base/java.lang.Thread.run(Unknown Source)
graylog1-1  | Caused by: java.lang.NullPointerException: Null sysInfo
graylog1-1  | 	at org.graylog2.system.stats.mongo.AutoValue_BuildInfo.<init>(AutoValue_BuildInfo.java:58)
graylog1-1  | 	at org.graylog2.system.stats.mongo.BuildInfo.create(BuildInfo.java:80)
graylog1-1  | 	at org.graylog2.system.stats.mongo.MongoProbe.createBuildInfo(MongoProbe.java:118)
graylog1-1  | 	at org.graylog2.system.stats.mongo.MongoProbe.<init>(MongoProbe.java:59)
graylog1-1  | 	at org.graylog2.system.stats.mongo.MongoProbe.<init>(MongoProbe.java:53)
graylog1-1  | 	at org.graylog2.system.stats.mongo.MongoProbe$$FastClassByGuice$$ffffffff8c038ed6.GUICE$TRAMPOLINE(<generated>)
graylog1-1  | 	at org.graylog2.system.stats.mongo.MongoProbe$$FastClassByGuice$$ffffffff8c038ed6.apply(<generated>)
graylog1-1  | 	at com.google.inject.internal.DefaultConstructionProxyFactory$FastClassProxy.newInstance(DefaultConstructionProxyFactory.java:82)
graylog1-1  | 	at com.google.inject.internal.ConstructorInjector.provision(ConstructorInjector.java:114)
graylog1-1  | 	at com.google.inject.internal.ConstructorInjector.construct(ConstructorInjector.java:91)
graylog1-1  | 	at com.google.inject.internal.ConstructorBindingImpl$Factory.get(ConstructorBindingImpl.java:300)
graylog1-1  | 	at com.google.inject.internal.ProviderToInternalFactoryAdapter.get(ProviderToInternalFactoryAdapter.java:40)
graylog1-1  | 	at com.google.inject.internal.SingletonScope$1.get(SingletonScope.java:169)
graylog1-1  | 	at com.google.inject.internal.InternalFactoryToProviderAdapter.get(InternalFactoryToProviderAdapter.java:45)
graylog1-1  | 	at com.google.inject.internal.SingleParameterInjector.inject(SingleParameterInjector.java:40)
graylog1-1  | 	at com.google.inject.internal.SingleParameterInjector.getAll(SingleParameterInjector.java:60)
graylog1-1  | 	at com.google.inject.internal.ConstructorInjector.provision(ConstructorInjector.java:113)
graylog1-1  | 	at com.google.inject.internal.ConstructorInjector.construct(ConstructorInjector.java:91)
graylog1-1  | 	at com.google.inject.internal.ConstructorBindingImpl$Factory.get(ConstructorBindingImpl.java:300)
graylog1-1  | 	at com.google.inject.internal.ProviderToInternalFactoryAdapter.get(ProviderToInternalFactoryAdapter.java:40)
graylog1-1  | 	at com.google.inject.internal.SingletonScope$1.get(SingletonScope.java:169)
graylog1-1  | 	at com.google.inject.internal.InternalFactoryToProviderAdapter.get(InternalFactoryToProviderAdapter.java:45)
graylog1-1  | 	at com.google.inject.internal.SingleParameterInjector.inject(SingleParameterInjector.java:40)
graylog1-1  | 	at com.google.inject.internal.SingleParameterInjector.getAll(SingleParameterInjector.java:60)
graylog1-1  | 	at com.google.inject.internal.ConstructorInjector.provision(ConstructorInjector.java:113)
graylog1-1  | 	at com.google.inject.internal.ConstructorInjector.construct(ConstructorInjector.java:91)
graylog1-1  | 	at com.google.inject.internal.ConstructorBindingImpl$Factory.get(ConstructorBindingImpl.java:300)
graylog1-1  | 	at com.google.inject.internal.InjectorImpl$1.get(InjectorImpl.java:1148)
graylog1-1  | 	... 43 more
graylog1-1  | MultiException stack 2 of 3
graylog1-1  | java.lang.IllegalArgumentException: While attempting to resolve the dependencies of org.graylog2.rest.resources.system.debug.bundle.SupportBundleResource errors were found
graylog1-1  | 	at org.jvnet.hk2.internal.ClazzCreator.resolveAllDependencies(ClazzCreator.java:247)
graylog1-1  | 	at org.jvnet.hk2.internal.ClazzCreator.create(ClazzCreator.java:387)
graylog1-1  | 	at org.jvnet.hk2.internal.SystemDescriptor.create(SystemDescriptor.java:479)
graylog1-1  | 	at org.glassfish.jersey.inject.hk2.RequestContext.findOrCreate(RequestContext.java:59)
graylog1-1  | 	at org.jvnet.hk2.internal.Utilities.createService(Utilities.java:2111)
graylog1-1  | 	at org.jvnet.hk2.internal.ServiceLocatorImpl.internalGetService(ServiceLocatorImpl.java:775)
graylog1-1  | 	at org.jvnet.hk2.internal.ServiceLocatorImpl.internalGetService(ServiceLocatorImpl.java:738)
graylog1-1  | 	at org.jvnet.hk2.internal.ServiceLocatorImpl.getService(ServiceLocatorImpl.java:708)
graylog1-1  | 	at org.glassfish.jersey.inject.hk2.AbstractHk2InjectionManager.getInstance(AbstractHk2InjectionManager.java:160)
graylog1-1  | 	at org.glassfish.jersey.inject.hk2.ImmediateHk2InjectionManager.getInstance(ImmediateHk2InjectionManager.java:30)
graylog1-1  | 	at org.glassfish.jersey.internal.inject.Injections.getOrCreate(Injections.java:122)
graylog1-1  | 	at org.glassfish.jersey.server.model.MethodHandler$ClassBasedMethodHandler.getInstance(MethodHandler.java:260)
graylog1-1  | 	at org.glassfish.jersey.server.internal.routing.PushMethodHandlerRouter.apply(PushMethodHandlerRouter.java:51)
graylog1-1  | 	at org.glassfish.jersey.server.internal.routing.RoutingStage._apply(RoutingStage.java:86)
graylog1-1  | 	at org.glassfish.jersey.server.internal.routing.RoutingStage._apply(RoutingStage.java:89)
graylog1-1  | 	at org.glassfish.jersey.server.internal.routing.RoutingStage._apply(RoutingStage.java:89)
graylog1-1  | 	at org.glassfish.jersey.server.internal.routing.RoutingStage._apply(RoutingStage.java:89)
graylog1-1  | 	at org.glassfish.jersey.server.internal.routing.RoutingStage._apply(RoutingStage.java:89)
graylog1-1  | 	at org.glassfish.jersey.server.internal.routing.RoutingStage.apply(RoutingStage.java:69)
graylog1-1  | 	at org.glassfish.jersey.server.internal.routing.RoutingStage.apply(RoutingStage.java:38)
graylog1-1  | 	at org.glassfish.jersey.process.internal.Stages.process(Stages.java:173)
graylog1-1  | 	at org.glassfish.jersey.server.ServerRuntime$1.run(ServerRuntime.java:266)
graylog1-1  | 	at org.glassfish.jersey.internal.Errors$1.call(Errors.java:248)
graylog1-1  | 	at org.glassfish.jersey.internal.Errors$1.call(Errors.java:244)
graylog1-1  | 	at org.glassfish.jersey.internal.Errors.process(Errors.java:292)
graylog1-1  | 	at org.glassfish.jersey.internal.Errors.process(Errors.java:274)
graylog1-1  | 	at org.glassfish.jersey.internal.Errors.process(Errors.java:244)
graylog1-1  | 	at org.glassfish.jersey.process.internal.RequestScope.runInScope(RequestScope.java:266)
graylog1-1  | 	at org.glassfish.jersey.server.ServerRuntime.process(ServerRuntime.java:253)
graylog1-1  | 	at org.glassfish.jersey.server.ApplicationHandler.handle(ApplicationHandler.java:696)
graylog1-1  | 	at org.glassfish.jersey.grizzly2.httpserver.GrizzlyHttpContainer.service(GrizzlyHttpContainer.java:367)
graylog1-1  | 	at org.glassfish.grizzly.http.server.HttpHandler$1.run(HttpHandler.java:190)
graylog1-1  | 	at com.codahale.metrics.InstrumentedExecutorService$InstrumentedRunnable.run(InstrumentedExecutorService.java:259)
graylog1-1  | 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
graylog1-1  | 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
graylog1-1  | 	at java.base/java.lang.Thread.run(Unknown Source)
graylog1-1  | MultiException stack 3 of 3
graylog1-1  | java.lang.IllegalStateException: Unable to perform operation: resolve on org.graylog2.rest.resources.system.debug.bundle.SupportBundleResource
graylog1-1  | 	at org.jvnet.hk2.internal.ClazzCreator.create(ClazzCreator.java:422)
graylog1-1  | 	at org.jvnet.hk2.internal.SystemDescriptor.create(SystemDescriptor.java:479)
graylog1-1  | 	at org.glassfish.jersey.inject.hk2.RequestContext.findOrCreate(RequestContext.java:59)
graylog1-1  | 	at org.jvnet.hk2.internal.Utilities.createService(Utilities.java:2111)
graylog1-1  | 	at org.jvnet.hk2.internal.ServiceLocatorImpl.internalGetService(ServiceLocatorImpl.java:775)
graylog1-1  | 	at org.jvnet.hk2.internal.ServiceLocatorImpl.internalGetService(ServiceLocatorImpl.java:738)
graylog1-1  | 	at org.jvnet.hk2.internal.ServiceLocatorImpl.getService(ServiceLocatorImpl.java:708)
graylog1-1  | 	at org.glassfish.jersey.inject.hk2.AbstractHk2InjectionManager.getInstance(AbstractHk2InjectionManager.java:160)
graylog1-1  | 	at org.glassfish.jersey.inject.hk2.ImmediateHk2InjectionManager.getInstance(ImmediateHk2InjectionManager.java:30)
graylog1-1  | 	at org.glassfish.jersey.internal.inject.Injections.getOrCreate(Injections.java:122)
graylog1-1  | 	at org.glassfish.jersey.server.model.MethodHandler$ClassBasedMethodHandler.getInstance(MethodHandler.java:260)
graylog1-1  | 	at org.glassfish.jersey.server.internal.routing.PushMethodHandlerRouter.apply(PushMethodHandlerRouter.java:51)
graylog1-1  | 	at org.glassfish.jersey.server.internal.routing.RoutingStage._apply(RoutingStage.java:86)
graylog1-1  | 	at org.glassfish.jersey.server.internal.routing.RoutingStage._apply(RoutingStage.java:89)
graylog1-1  | 	at org.glassfish.jersey.server.internal.routing.RoutingStage._apply(RoutingStage.java:89)
graylog1-1  | 	at org.glassfish.jersey.server.internal.routing.RoutingStage._apply(RoutingStage.java:89)
graylog1-1  | 	at org.glassfish.jersey.server.internal.routing.RoutingStage._apply(RoutingStage.java:89)
graylog1-1  | 	at org.glassfish.jersey.server.internal.routing.RoutingStage.apply(RoutingStage.java:69)
graylog1-1  | 	at org.glassfish.jersey.server.internal.routing.RoutingStage.apply(RoutingStage.java:38)
graylog1-1  | 	at org.glassfish.jersey.process.internal.Stages.process(Stages.java:173)
graylog1-1  | 	at org.glassfish.jersey.server.ServerRuntime$1.run(ServerRuntime.java:266)
graylog1-1  | 	at org.glassfish.jersey.internal.Errors$1.call(Errors.java:248)
graylog1-1  | 	at org.glassfish.jersey.internal.Errors$1.call(Errors.java:244)
graylog1-1  | 	at org.glassfish.jersey.internal.Errors.process(Errors.java:292)
graylog1-1  | 	at org.glassfish.jersey.internal.Errors.process(Errors.java:274)
graylog1-1  | 	at org.glassfish.jersey.internal.Errors.process(Errors.java:244)
graylog1-1  | 	at org.glassfish.jersey.process.internal.RequestScope.runInScope(RequestScope.java:266)
graylog1-1  | 	at org.glassfish.jersey.server.ServerRuntime.process(ServerRuntime.java:253)
graylog1-1  | 	at org.glassfish.jersey.server.ApplicationHandler.handle(ApplicationHandler.java:696)
graylog1-1  | 	at org.glassfish.jersey.grizzly2.httpserver.GrizzlyHttpContainer.service(GrizzlyHttpContainer.java:367)
graylog1-1  | 	at org.glassfish.grizzly.http.server.HttpHandler$1.run(HttpHandler.java:190)
graylog1-1  | 	at com.codahale.metrics.InstrumentedExecutorService$InstrumentedRunnable.run(InstrumentedExecutorService.java:259)
graylog1-1  | 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
graylog1-1  | 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
graylog1-1  | 	at java.base/java.lang.Thread.run(Unknown Source)
graylog1-1  | 
graylog1-1  | `

## Steps to Reproduce (for bugs)
<!--- Provide a link to a live example, or an unambiguous set of steps to -->
<!--- reproduce this bug. Include code to reproduce, if relevant -->
1. Install Graylog (potentially from [here](https://github.com/fjagwitz/Graylog-Cookbooks/tree/main/01_Installation))
2. Change the MongoDB version from v7 to v8
3. Login through the WebUI and try to create a Support Bundle
4. Observe that it does not work :-( 

Tested this with Graylog 6.3.3, 7.0.0, 7.0.1 and 7.0.2 (all versions don't create Support Bundles with MongoDB v8 in the backend)

## Your Environment
<!--- Include as many relevant details about the environment you experienced the bug in -->

* Graylog Version: v7
* Java Version: -
* OpenSearch Version: 2.19.3
* MongoDB Version: 8
* Operating System: Ubuntu/Docker

--- a/changelog/unreleased/issue-24581.toml
+++ b/changelog/unreleased/issue-24581.toml
@@ -1,0 +1,5 @@
+type = "fixed"
+message = "Fix compatibility issue with MongoDB >=8.2."
+
+issues = ["24581"]
+pulls = ["24733"]

--- a/graylog2-server/src/main/java/org/graylog2/system/stats/mongo/BuildInfo.java
+++ b/graylog2-server/src/main/java/org/graylog2/system/stats/mongo/BuildInfo.java
@@ -36,6 +36,7 @@ public abstract class BuildInfo {
     public abstract String gitVersion();
 
     @JsonProperty("sys_info")
+    @Nullable
     public abstract String sysInfo();
 
     @JsonProperty("loader_flags")

